### DISCUSSION TITLE
Exclude `node_modules` from Time Machine backups (macOS)

### DISCUSSION THREAD

**Discussion by kasper9n** [npm/feedback#570] [CLI] [answered]
`node_modules` can contain a lot of files and change frequently, which can significantly slow down backup solutions like Time Machine, so I think it would make sense to exclude them from backups. This can be done by adding the xattr `com.apple.metadata:com_apple_backup_excludeItem` to the folders.

Rust already does this for their `target` folders: https://github.com/rust-lang/cargo/pull/4386

**Comment by MylesBorins** [comment, marked as answer]
If you would like to propose this as a change to the npm CLI you should open an issue over at https://github.com/npm/rfcs

**Reply by kasper9n** [reply]
Okay I'll do that, thanks!
